go.mod

```diff
@@ -1,89 +1,81 @@
 module github.com/cloudnativelabs/kube-router/v2
```

Comment:
It looks like the rebase here, favored your older library versions instead of the ones that were updated by dependabot. We should fix this up to favor the upgraded library versions (as well as Go 1.25).